packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/InteractionLayer.js

```diff
@@ -87,6 +90,30 @@ function InteractionLayer({
     return svgEventOffset
   }
 
+  // Get the Subject's Caesar Reductions for FreehandLine Tool
+  useEffect(() => {
+    SubjectReductions({
+      workflowId: subject?.workflow.id,
+      subjectId: subject?.id,
+      cb: (data) => {
+        if (data) {
+          const coors = data[0].data.points.map(val => {
+            return { x: val[0], y: val[1] };
+          });
+
+          const mark = activeTool.createMark({
+            id: cuid(),
+            frame: 0,
+            toolIndex: 0,
+          }, coors)
+
+          setActiveMark(mark)
+          setCreating(true)
+        }
+      }
+    })
+  }, []);
```

Comment:
This effect has [undeclared dependencies](https://beta.reactjs.org/reference/react/useEffect#specifying-reactive-dependencies).

Reply:
It will also run for all projects and workflows, which means that you can't guarantee that the active tool will be an editable freehand line. That might cause side effects for transcription projects, which also use `activeTool` and `setActiveMark`.

Reply:
I’ve started work in #4269 to extend the existing `Subject` model, so that it works with any reducer key, not just ‘Alice’.

Reply:
This code has been removed to incorporate the hook plus above changes suggested above in a separate PR